assignments/callbacks.js

```diff
@@ -19,20 +19,29 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
     console.log(first)
   });
 
-*/
-
-
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.getLength);
```

Comment:
```suggestion
  return cb(arr.length);
```
the method you are looking for is arr.length